src/guide/mailing-lists.md

```diff
@@ -10,6 +10,10 @@ The mailing lists are the key communications mechanism for all OpenJDK work. All
 
 > [`mail.openjdk.org`](https://mail.openjdk.org/mailman/listinfo)
 
+:::{.note}
+Please note that OpenJDK is about **development** of the JDK, not usage. This is not a place to ask support questions. Please direct those to whoever provided your JDK build.
```

Comment:
Nit: suggest changing to "to whoever" to "to whomever".

More substantive, I think a link to the public bug tracker where issues can be filed would be helpful here too.

Reply:
Fixed.